cosmos-db/mongo-init.js

```diff
@@ -0,0 +1,2021 @@
+  getLocalCouncilsFailed: 0,
+  mostRecentGetLocalCouncilsSucceeded: true,
+  getCouncilDataFailed: 0,
+  getCouncilDataSucceeded: 0,
+  mostRecentGetCouncilDataSucceeded: true,
+});
+
+db = db.getSiblingDB("config");
+db.createUser({
+  user: "test-user",
+  pwd: "test-password",
+  roles: [
+    {
+      role: "root",
+      db: "admin",
+    },
+  ],
+});
+
+db.createCollection("localAuthorities");
+db.localAuthorities.insertOne({
+  _id: 8015,
+  local_council: "Fakes Council",
+  local_council_email: "fakecouncil@test.com",
+  local_council_notify_emails: ["LC1_fsatestemail.valid@gmail.com"],
+  local_council_phone_number: "01234 456788",
+  local_council_url: "fakes",
+  country: "england",
+});
+
+db.localAuthorities.insertOne({
+  _id: 4226,
+  local_council: "Dorset County Council",
+  local_council_email: "fsatestemail.valid@gmail.com",
+  local_council_notify_emails: ["fsatestemail.valid@gmail.com"],
+  local_council_phone_number: "123456789",
+  local_council_url: "",
+  country: "england",
+  auth: {
+    url: "https://fsa-pilot-enviro.tascomi.com/rest/v1/",
+    public_key: "changeme",
+    private_key: "changeme",
+  },
+});
+
+db.localAuthorities.insertOne({
+  _id: 4221,
+  local_council: "West Dorset District Council",
+  local_council_email: "fsatestemail.valid@gmail.com",
+  separate_standards_council: 4226,
+  local_council_notify_emails: ["LC1_fsatestemail.valid@gmail.com"],
+  local_council_phone_number: "02388 899766",
+  local_council_url: "west-dorset",
+  country: "england",
+  auth: {
+    public_key: "changeme",
+    private_key: "changeme",
+    url: "https://fsa-pilot-enviro.tascomi.com/rest/v1/",
+  },
+});
+
+db.localAuthorities.insertOne({
+  _id: 4012,
+  local_council: "Fakes cardiff council",
+  local_council_email: "fakecouncilcardiff@test.com",
+  local_council_notify_emails: ["LC1_fsatestemail.validcardiff@gmail.com"],
+  local_council_phone_number: "01233 333445",
+  local_council_url: "cardiff",
+  country: "england",
+});
+
+db.localAuthorities.insertOne({
+  _id: 4013,
+  local_council: "Fakes vale of G council",
+  local_council_email: "fakecouncilvog@test.com",
+  local_council_notify_emails: ["LC1_fsatestemail.validvog@gmail.com"],
+  local_council_phone_number: "01233 333445",
+  local_council_url: "the-vale-of-glamorgan",
+  country: "england",
+});
+db.localAuthorities.insertOne({
+  _id: 6008,
+  local_council: "Mid & East Antrim Borough Council",
+  local_council_email: "fsatestemail.valid@gmail.com",
+  local_council_notify_emails: ["fsatestemail.valid@gmail.com"],
+  local_council_phone_number: "123456789",
+  local_council_url: "mid-and-east-antrim",
+  country: "northern-ireland",
+});
+
+db.createCollection("suppliers");
+db.suppliers.insertOne({
+  _id: 1,
+  supplier_name: "Test Supplier",
+  supplier_url: "test-supplier",
+  local_council_urls: ["cardiff", "bath"],
+});
+
+db.createCollection("configVersion");
+db.configVersion.insertOne({
+  _id: "1.7.0",
```

Comment:
Worth updating this to latest prod configVersion number?

Reply:
Would also need to update the FE .env to match

Reply:
Done